lib/pbench/server/database/models/users.py

```diff
@@ -131,16 +133,22 @@ def delete(username):
             Database.db_session.rollback()
             raise
 
-    def is_admin(self):
+    def is_admin(self) -> bool:
         """This method checks whether the given user has an admin role.
-        This can be extended to groups as well for example a user belonging to certain group has only those
-        privileges that are assigned to the group.
+
+        This can be extended to groups as well for example a user belonging to
+        certain group has only those privileges that are assigned to the
+        group.
+
+        Returns:
+            True if the user's role is ADMIN, False otherwise.
         """
         return self.role is Roles.ADMIN
 
     @staticmethod
-    def is_admin_username(username):
-        # TODO: Need to add an interface to fetch admins list instead of hard-coding the names, preferably via sql query
+    def is_admin_username(username) -> bool:
+        # TODO: Need to add an interface to fetch admins list instead of
+        # hard-coding the names, preferably via sql query.
```

Comment:
If you're going to wrap this, could you indent the second line, like:
```
        # TODO: Need to add an interface to fetch admins list instead of
        #       hard-coding the names, preferably via sql query.
```
I don't know about your IDE, but mine color-highlights things like `TODO`'s, but it will treat line 151 as a new comment -- and not highlight it -- if it's not internally indented.

Reply:
Shouldn't this be a *docstring* rather than a comment?? (Yeah, I know; you didn't create it, but if you weren't going to edit it already ...???)

Reply:
I'll move it to a doc string, but do we want to keep a TODO in this form if we are going to do that?

Reply:
Fixed, see if you like what I did.